app/config/sources.inc.php

```diff
@@ -185,11 +185,28 @@
     'teach/smarton/security.lang',
     'teach/smarton/surveillance.lang',
     'teach/smarton/tracking.lang',
+    'thunderbird/channel.lang',
     'thunderbird/features.lang',
     'thunderbird/index.lang',
     'thunderbird/start/release.lang',
 ];
 
+/*
+    Thundebird locales on Release channel
+    Source: http://hg.mozilla.org/releases/comm-release/raw-file/tip/mail/locales/shipped-locales
+    This variable is used in the $lang_flags['www.mozilla.org'] array below,
+    so it must be defined before it.
+*/
+$thunderbird_release = [
+    'ar', 'ast', 'az', 'be', 'bg', 'bn-BD', 'br', 'ca', 'cs', 'cy',
+    'da', 'de', 'dsb', 'el', 'en-GB', 'es-AR', 'es-ES', 'et',
+    'eu', 'fi', 'fr', 'fy-NL', 'ga-IE', 'gd', 'gl', 'he', 'hr',
+    'hsb', 'hu', 'hy-AM', 'id', 'is', 'it', 'ja', 'ko', 'lt',
+    'nb-NO', 'nl', 'nn-NO', 'pa-IN', 'pl', 'pt-BR', 'pt-PT',
+    'rm', 'ro', 'ru', 'si', 'sk', 'sl', 'sq', 'sr', 'sv-SE',
+    'tr', 'uk', 'vi', 'zh-CN', 'zh-TW',
+];
```

Comment:
At this point I would prefer moving it directly to locales.inc.php, it makes things a little clearer (and maybe `thunderbird_locales` instead of `thunderbird_release`).

Besides that, feel free to merge